### ISSUE TITLE
How to use this plugin and deploy the site to Github in User / Organization mode

### ISSUE THREAD

**Issue by Harrypotterrrr** [linjer/jekyll-katex#24]
Thank you for sharing with this plugin, but as I deployed this site to Github, it told me `Liquid syntax error: Unknown tag 'katex`. I have checked this issue and found out that the official whitelist excludes the third-party plugins for the sake of safety, just like `jekyll-katex`.

There is a workaround to deal with this problem is that change the name of the repo to others except for `<username>.github.io`, accordingly changing the site from User organization mode to project mode. However, I want to make Github build this code for me, and save the permalink `<username>.github.io` instead of `<username>.github.io/<repo-name>`. 

In summary, I wonder if there is any way I can use your plugins as well as stay in User/Organization mode?

**Comment by linjer** [comment]
I haven't looked into this aspect yet. I can add this to the backlog or if you have a PR for fixing this, I could review that.
